infra/modules/providers/azure/service-plan/variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "name" {
+  description = "(Required) Specifies the human consumable label for this resource."
```

Comment:
you may want to mention which resource names will be randomized in the absence of `name`

Reply:
This is provided in the output after the resources are created.